rootfs/etc/profile.d/_01-launch-warning.sh

```
@@ -0,0 +1,59 @@
+# Specifically, this guards against:
+#   docker run -it cloudposse/geodesic:latest-debian  | bash
+
+# If the warning message is longer than the terminal width,
+# on some terminals the message may be truncated. In that case,
+# all the erasure characters will not be printed, and the message will not be erased.
+# So we take pains to make the printed message appear short on the terminal.
+# We do that in part by using backspaces to erase the message one character at a time,
+# so the cursor never advances. Then we eval the message with backspaces removed.
+# We have to then add some extra characters to erase the eval command.
+function warn_if_piped() {
+	local saved_stty=$(stty -g)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)